Content.Server/ParticleAccelerator/Components/ParticleAcceleratorControlBoxComponent.cs

```diff
@@ -167,6 +168,18 @@ public sealed partial class ParticleAcceleratorControlBoxComponent : Component
     [ViewVariables]
     public bool StrengthLocked = false;
 
+    /// <summary>
+    /// Time at which the admin alarm sound effect can next be played.
+    /// </summary>
+    [DataField("effectCooldown", customTypeSerializer: typeof(TimeOffsetSerializer))]
```

Comment:
you don't have to write `[DataField("effectCooldown",` its outdated format
just `[DataField(custom...`